contrib/common.sh

```diff
@@ -61,6 +61,61 @@ function setup_ruby_gem() {
   echo 'install: --no-format-executable' >> /etc/gemrc
 }
 
+function setup_kerberos_server() {
```

Comment:
This seems to be for setting up a server on the OBS instance. I don't think we want this. The kerberos server should be on a different host. Otherwise we could just use the existing login / password authentification